src/main/java/com/pinterest/secor/common/KafkaClient.java

```diff
@@ -141,10 +144,7 @@ private Message getMessage(TopicPartition topicPartition, long offset,
             payloadBytes = new byte[payload.limit()];
             payload.get(payloadBytes);
         }
-        Long timestamp = null;
```

Comment:
long timestamp = 0l;
        if (mConfig.useKafkaTimestamp()) {
            timestamp = mKafkaMessageTimestampFactory.getKafkaMessageTimestamp().getTimestamp(kafkaMessage);
        }

Is this fine?